_guide/_pages/python.md

```diff
@@ -63,6 +63,8 @@ isort](https://pypi.python.org/pypi/flake8-isort) for import order. We
 ecosystem](https://pypi.python.org/pypi?%3Aaction=search&term=flake8&submit=search)
 for more functionality.
 
+Use [Black](https://black.readthedocs.io/en/stable/) for automatic code formatting.
```

Comment:
I don't have strong feelings about black vs. [alternatives](https://docs.python-guide.org/writing/style/#auto-formatting), but I get the impression black is the most opinionated, and is thus the best 😏

Reply:
From the Python formatters I've tried, Black is the first that doesn't allow customization of the actual style guide in a config file (e.g. setting "max-line-length" to something someone on a team deems reasonable for them).

I agree with the broad recommendation of Black for most projects.

Reply:
@afeld Flake8's default settings (mentioned a few lines above) are inconsistent with Black's formatting in a couple cases. See https://black.readthedocs.io/en/stable/compatible_configs.html#flake8

Recommended flake8 settings are:
```
[flake8]
max-line-length = 88
extend-ignore = E203, W503
```